BootloaderCorePkg/Stage2/Stage2.c

```diff
@@ -362,8 +363,9 @@ S3ResumePath (
 
   AddMeasurePoint (0x31F0);
 
-  // Stop timer so boot retries stop
-  if (PcdGetBool (PcdSblResiliencyEnabled)) {
+  // FWU payload is the only payload in SBL scope, so stop TCO
+  // timer if another payload is set to be launched
+  if (PcdGetBool (PcdSblResiliencyEnabled) && GetBootMode () != BOOT_ON_FLASH_UPDATE) {
```

Comment:
in S3 resume path, there is no payload would be run including Osloader and UEFI payload.
in this case, we should StopTcoTimer too I think.